flake.nix

```diff
@@ -57,7 +57,7 @@
 
           # we also want cross compilation to windows on linux (and only with default compiler).
           crossPlatforms = p:
-            lib.optional (system == "x86_64-linux" && config.compiler-nix-name == defaultCompiler)
+            lib.optional (system == "x86_64-linux" && config.compiler-nix-name == "ghc964")
```

Comment:
can you make another variable in line 51 with the version of mingw compiler and re-use it? just to make it easier to find out which ghc versions are we building

Reply:
I put it higher in the file in the end, because `defaultCompiler` variable moved in `main`, but it is done in https://github.com/IntersectMBO/cardano-cli/pull/791/commits/30312f6ef847c1d80f29a9268128184c84b6ef1c